provider/core/src/export/mod.rs

```diff
@@ -50,14 +50,14 @@ pub trait DataExporter: Sync {
     /// markers have been fully dumped). This conceptually takes ownership, so
     /// clients *may not* interact with this object after close has been called.
     fn close(&mut self) -> Result<ExporterCloseMetadata, DataError> {
-        Ok(ExporterCloseMetadata)
+        Ok(ExporterCloseMetadata::default())
     }
 }
 
-#[non_exhaustive]
-#[derive(Debug, Clone, Default)]
+#[derive(Debug, Default)]
+#[allow(clippy::exhaustive_structs)] // newtype
 /// Contains information about a successful export.
-pub struct ExporterCloseMetadata;
+pub struct ExporterCloseMetadata(pub Option<Box<dyn core::any::Any>>);
```

Comment:
thought: dyn Display or Debug? Could be paired with Any.

Reply:
I don't think there's much of a point just printing everything that's in there